client/web/compose/src/views/Public/Pages/Records/View.vue

```diff
@@ -146,6 +158,23 @@ export default {
 
       return this.$t(`page:public.record.${titlePrefix}.title`, { name: name || handle, interpolation: { escapeValue: false } })
     },
+
+    recordListModule () {
+      if (this.page.moduleID) {
+        return this.getModuleByID(this.page.moduleID)
+      } else {
+        return undefined
+      }
+    },
+
+    nextPrevNavigationOptions () {
+      const index = this.recordIdx !== undefined ? this.recordIdx : 1
```

Comment:
How about having a getter in store again, that always just returns a boolean of next or previous item. And then you can just return an object with those two getters.